src/components/appheader.main.jsx

```diff
@@ -124,35 +127,38 @@ class AppHeaderMain extends React.Component {
 
           <div className="central-container">
             <div className="horizontal-menu">
+              {availability && (
```

Comment:
We shouldn't hide the navigation altogether here. The `CATALOG_BROWSER` role has access to browse the catalog, but cannot add to cart or wishlists. So on the Product Display Page, we should disable those buttons if the forms are not returned by the API.